ledger/apply/application_test.go

```diff
@@ -37,22 +38,14 @@ import (
 // call clone on each basics.AppParams -- callers must do that for any values where
 // they intend to modify a contained reference type.
 func cloneAppParams(m map[basics.AppIndex]basics.AppParams) map[basics.AppIndex]basics.AppParams {
-	res := make(map[basics.AppIndex]basics.AppParams, len(m))
-	for k, v := range m {
-		res[k] = v
-	}
-	return res
+	return maps.Clone(m)
 }
 
 // Allocate the map of LocalStates if it is nil, and return a copy. We do *not*
 // call clone on each AppLocalState -- callers must do that for any values
 // where they intend to modify a contained reference type.
 func cloneAppLocalStates(m map[basics.AppIndex]basics.AppLocalState) map[basics.AppIndex]basics.AppLocalState {
-	res := make(map[basics.AppIndex]basics.AppLocalState, len(m))
-	for k, v := range m {
-		res[k] = v
-	}
-	return res
+	return maps.Clone(m)
```

Comment:
both `cloneAppParams` and `cloneAppLocalStates` seem to be used 2 times in this file, guess we can go further by taking these 2 down and replace both usages with `maps.Clone`?

Reply:
I was tempted to do that - since the function is now one line, it seems kind of pointless.

Do any other reviewers like or dislike such a change?

Reply:
I'm in favor of removing the function

Reply:
Two votes in favor. Sold!